src/send/Send.tsx

```diff
@@ -127,7 +129,9 @@ function Send({ route }: Props) {
 
     // Only show currency picker once we know that the recipient is verified,
     // and only if the user is permitted to change tokens.
-    if (defaultTokenIdOverride) {
+    if (defaultTokenIdOverride || getFeatureGate(StatsigFeatureGates.USE_NEW_SEND_FLOW)) {
```

Comment:
the gate can just go in a separate `if` block right? E.g.,
```ts
if (getFeatureGate(StatsigFeatureGates.USE_NEW_SEND_FLOW)) {
  navigate(new send screen) // for now could be old screen
}
else if (defaultTokenOverride) {
  navigate(old send amount)
}
// show bottom sheet
```

Reply:
Currently they both go to the same place, but I think having the separate if matches the A/C better: fixed in e553865f470b40710327a1724e59ab2c1cd8af3a!